examples/Vanilla/index.html

```diff
@@ -13,6 +13,8 @@ <h1>Examples</h1>
     <li><a href="OnLoad/index.html">On Load Page</a></li>
     <li><a href="Error/index.html">Error Page</a></li>
     <li><a href="CustomCollector/index.html">Custom Collector Page</a></li>
+    <li><a href="WebBasedCollectorsCollection/index.html">Web Based Collectors Collection Page</a></li>
+    <li><a href="WebBasedCollectorsCollection/index.html">Web Based Collectors Page</a></li>
```

Comment:
why these are identical? shouldn't it be WebBasedCollector and WebBasedCollectorCollection?

Reply:
One of them is for when you use them manually one by one. The other one is for when you want to use a pre made collection with all the web focused ones.